app/controllers/articles_controller.rb

```diff
@@ -6,6 +6,8 @@ def index
 
   def show
     @article = Article.find(params[:id])
+    @comments = @article.comments.order(created_at: :desc).limit(20)
```

Comment:
If you are going to limit how many comments you are showing by default, you should add some sort of pagination around the comments, so I would either do that or just straight up remove this limit.